### DISCUSSION TITLE
I have verified as a teacher but haven't got my benefits like Co-Pilot on my account.

### DISCUSSION THREAD

**Discussion by zionmezba** [community/community#145415] [GitHub Education] [answered]
### Hi everyone,

I have verified as a teacher but haven't got my benefits like Co-Pilot on my account. What can I do? Any Suggestion?

![image](https://github.com/user-attachments/assets/d6a9f893-12c8-4f4a-aa22-76cadcff9357)

![image](https://github.com/user-attachments/assets/276bb660-9368-4af5-96aa-29a559e28f75)

**Comment by davevad93** [comment, marked as answer]
Hi @zionmezba ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @zionmezba was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
